src/app/components/case/tasks/ChangeDueDate/ChangebleDueDate.tsx

```diff
@@ -39,8 +40,8 @@ const ChangeableDueDate: React.FC<Props> = ({ dueDate, camundaTaskId, caseId })
 
   const [, { execPost }] = useDueDate({ lazy: true })
   const onSubmit = (data: any) => {
-    console.log(data, camundaTaskId, caseId)
-    execPost()
+    if( data.date) { data.date = appendTimeToDate(data.date) }
```

Comment:
Maybe nicer to handle this in a `mapSubmitData` function:
```
const mapSubmitData = (data: Components.Schemas.CamundaDateUpdate) => ({
  ...data,
  date: appendTimeToDate(data.date)
})
```
Also think date can not be `undefined`.

Reply:
Fixed it